blockstack/blockstackd.py

```diff
@@ -1433,6 +1433,11 @@ def export_account_state(self, account_state):
             'vtxindex': account_state['vtxindex'],
             'txid': account_state['txid'],
         }
+        # if block height is after the migration export threshold, return a lock height that will force the wallet to error when sending a tx
+        if result['block_id'] >= 665750:
```

Comment:
Can you use the constant IMPORT_BLOCK defined in Aaron's PR?

Reply:
idk how to import that from the other file. @kantai

Reply:
Not critical if that's not straight forward.

Reply:
Probably is but python module import is cancer